conf/sparkler-default.yaml

```diff
@@ -70,11 +73,19 @@ fetcher.server.delay: 1000
 # Discouraged to Modify unless specifically required.
 plugins.bundle.directory: ${project.parent.basedir}${file.separator}${project.bundles.directory}
 
+plugins.cache.directory: felix-cache
```

Comment:
As discussed, we don't need this anymore. I will remove this configuration property.